esm/shared/mime.js

```diff
@@ -6,26 +6,38 @@ export const Mime = {
   'text/html': {
     docType: '<!DOCTYPE html>',
     ignoreCase: true,
+    escapeHtmlEntities: true,
     voidElements: /^(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr)$/i
   },
   'image/svg+xml': {
     docType: '<?xml version="1.0" encoding="utf-8"?>',
     ignoreCase: false,
+    escapeHtmlEntities: true,
     voidElements
   },
   'text/xml': {
     docType: '<?xml version="1.0" encoding="utf-8"?>',
     ignoreCase: false,
+    escapeHtmlEntities: true,
     voidElements
   },
   'application/xml': {
     docType: '<?xml version="1.0" encoding="utf-8"?>',
     ignoreCase: false,
+    escapeHtmlEntities: true,
     voidElements
   },
   'application/xhtml+xml': {
     docType: '<?xml version="1.0" encoding="utf-8"?>',
     ignoreCase: false,
+    escapeHtmlEntities: true,
+    voidElements
+  },
+  'text/jsx+xml': {
+    docType: '<?xml version="1.0" encoding="utf-8"?>',
+    ignoreCase: false,
+    escapeHtmlEntities: false,
+    unquotedJsonAttributes: true,
```

Comment:
2 things I'd rather change here, or discuss:

  1. I don't like checking *emptiness* ... all objects here are homogeneous, if we add a single boolean field somewhere I want it to be explicitly boolean field somewhere else too, as it is for `escapeHtmlEntities` ... so, I'd add `unquotedJsonAttributes: flase` in other cases
  2. I don't unrestand why it's called `unquotedJsonAttributes` ... I understand it's virtually JSON, but practically that's an unquoted attribute ... so ... since we want to allow only JSON which results into being unquoted, I think a field name such as `jsonAttributes` would better reflect the intent/purpose? ... 'cause of course those gotta be unquoted, otherwise it'd be invalid JSON, does it make sense?

Reply:
1. agreed, that makes sense to me
2. I think that `jsonAttributes` would describe the purpose what should be achieved, while `unquotedJsonAttributes` is describing what it is happening. I like both ways of naming.  Lets go with `jsonAttributes` then